test/validators/movie.test.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
```

Comment:
You should have one test that actually succeeds -- something like `allows all` with every optional param.

Deciding what tests to do on validators is actually tricky, bc there are nearly limitless permutations you can test. I usually go with the following:

1. Allows everything
2. Some plausible errors that would severely mess things up if they happened
3. Any complex validator logic.

For for this, it's probably fine to have a test for everything as you have. But in big validators doing all that testing is often not the best use of time.